data/dev/design/ota/judge_prompts/design_telescopic_single_end.md

```
@@ -0,0 +1,28 @@
+
+{path:../../judge_prompts/design_ota_common_instructions.md}
+
+Scoring: compute `overall = {weight_structural_correctness}*scores.structural_correctness + {weight_topology_terms}*scores.topology_terms + {weight_safety}*scores.safety`.
+
+## Criteria
+
+{path:../../judge_prompts/design_ota_common_criteria.md}
+
+## Answer Key
+
+### structural_correctness
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)